github.sgnl.yaml

```diff
@@ -0,0 +1,1099 @@
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        relationshipAttributeId: Collaborator-id
+      - name: avatarUrl
+        externalId: avatarUrl
+        type: String
+      - name: bio
+        externalId: bio
+        type: String
+      - name: bioHTML
+        externalId: bioHTML
+        type: String
+      - name: company
+        externalId: company
+        type: String
+      - name: companyHTML
+        externalId: companyHTML
+        type: String
+      - name: createdAt
+        externalId: createdAt
+        type: DateTime
+      - name: databaseId
+        externalId: databaseId
+        type: Int64
+      - name: email
+        externalId: email
+        type: String
+      - name: login
+        externalId: login
+        type: String
+      - name: isBountyHunter
+        externalId: isBountyHunter
+        type: Bool
+      - name: isCampusExpert
+        externalId: isCampusExpert
+        type: Bool
+      - name: isDeveloperProgramMember
+        externalId: isDeveloperProgramMember
+        type: Bool
+      - name: isEmployee
+        externalId: isEmployee
+        type: Bool
+      - name: isFollowingViewer
+        externalId: isFollowingViewer
+        type: Bool
+      - name: isHireable
+        externalId: isHireable
+        type: Bool
+      - name: isSiteAdmin
+        externalId: isSiteAdmin
+        type: Bool
+      - name: isViewer
+        externalId: isViewer
+        type: Bool
+      - name: location
+        externalId: location
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: pinnedItemsRemaining
+        externalId: pinnedItemsRemaining
+        type: Int64
+      - name: projectsResourcePath
+        externalId: projectsResourcePath
+        type: String
+      - name: projectsUrl
+        externalId: projectsUrl
+        type: String
+      - name: resourcePath
+        externalId: resourcePath
+        type: String
+      - name: twitterUsername
+        externalId: twitterUsername
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+      - name: url
+        externalId: url
+        type: String
+      - name: viewerCanChangePinnedItems
+        externalId: viewerCanChangePinnedItems
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanFollow
+        externalId: viewerCanFollow
+        type: Bool
+      - name: viewerIsFollowing
+        externalId: viewerIsFollowing
+        type: Bool
+      - name: websiteUrl
+        externalId: websiteUrl
+        type: String
+relationships:
+  RepositoryToOrganization:
+    displayName: OrganizationRepository
+    name: MemberOf
+    fromAttribute: Repository-orgId 
```

Comment:
Is this formatting standard across different SoRs in our adapters? Wondering if the camelCase/naming conventions here should make this more like "repository-orgId" or just "repositoryOrgId." No strong stance here on the formatting, but just wanted to double check that this is consistent across this template and other SoRs.

Reply:
Let's decide on one of `lowerCamelCase` or `UpperCamelCase` here. `-`'s may cause issues down the line.

Reply:
will change to upper camel case